docs/sidebar.json

```diff
@@ -90,7 +97,7 @@
       ]
     }
   ],
-  "Debug & Help": ["debug/csrf"],
+  "Debug & Help": ["debug/csrf", "debug/argon2"],
```

Comment:
Incorrect ID

Reply:
damn, always forget the sidebar :facepalm: